cirrus/server/cirrus/settings.py

```diff
@@ -19,3 +19,4 @@
         "channel": channel,
     }
 )
+fml_path: str = cast(str, config("CIRRUS_FML_PATH", default=""))
```

Comment:
You could mock this in the test to override it in Python space to a test fixture path like here:

https://github.com/mozilla/experimenter/blob/main/experimenter/experimenter/features/tests/__init__.py#L6-L10